tests/test_api.py

```diff
@@ -140,6 +140,7 @@ async def test_get_event(
         start=DateOrDatetime(date=datetime.date(2022, 4, 13)),
         end=DateOrDatetime(date=datetime.date(2022, 4, 14)),
         transparency="transparent",
+        private_calendar_id="some-calendar-id",
```

Comment:
The `private_calendar_id` attribute is no longer excluded from pure equals checks. The "old" assert would probably pass if we do the roundtrip to json and back since `exclude=True` is still set.